src/reports/SearchList.jsx

```diff
@@ -17,9 +14,14 @@ class SearchList extends React.Component {
     this.searchInstitutions = this.searchInstitutions.bind(this)
   }
 
-  componentDidMount() {
+  getData() {
```

Comment:
Outside of adding the correct URL for 2018 when available, I think it would be good to include a caching mechanism for the institutions per year.

This would likely be similar to the INSTITUTIONS variable before, but it would just be an object that would store institutions per year.

```
const INSTITUTIONS = {
   2017: {},
   2018: {/*institutions go here*/}
}
```

Reply:
Yep, makes perfect sense to me. I'll update soon.